### ISSUE TITLE
IconFont not working

### ISSUE THREAD

**Issue by macjohson** [dotnet/maui#8426]
### cs 
```c#
public partial class NxIcon : ContentView
{
	public static readonly BindableProperty TypeProperty =
		BindableProperty.Create(
			nameof(Type),
			typeof(DeviceIconType),
			typeof(NxIcon),
			DeviceIconType.Ai
			);

	public DeviceIconType Type
	{
		get { return (DeviceIconType)GetValue(TypeProperty); }
                set {SetValue(TypeProperty, value);}
	}

	public NxIcon()
	{
		InitializeComponent();
	}
}
```

### Convert
```cs
public class IconEnumToIUnicode : IValueConverter
    {
        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
        {
            switch ((DeviceIconType)value)
            {
                case DeviceIconType.WindowsComputer:
                    return "&#xe882;";

                case DeviceIconType.AndroidMirrorAllInOne:
                    return "&#xe613;";

                ...
            }

            return "&#xe882;";
        }

        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
        {
            return DeviceIconType.Ai;
        }
    }
```

### used
```xaml
<controls:NxIcon Type="Ai"/>
```

### xaml

**working**
```xaml
<Label 
        FontFamily="IconFont" 
        Text="{Binding Type, Converter={StaticResource iconUnicodeConvert}}" 
        TextType="Html" 
        />
```

_**not working**_
just show unicode text
```xaml
<Image>
        <Image.Source>
            <FontImageSource 
                FontFamily="IconFont"
                Glyph="{Binding Type, Converter={StaticResource iconUnicodeConvert}}"
                />
        </Image.Source>
</Image>
```

_**not working**_
just show unicode text
```xaml
<Label FontFamily="IconFont" Text="{Binding Type, Converter={StaticResource iconUnicodeConvert}}"/>
```

![image](https://user-images.githubusercontent.com/16527096/176496950-718c4381-417c-41d9-95c5-5a4882a480ee.png)

### Steps to Reproduce

1. Create MAUI via vs2022 preview
2. add ContentView, custom NxIcon Control
3. add ttf file to resource fonts fold
4. add font via CreateMauiApp func

### Version with bug

6.0.400 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Android, Windows

### Affected platform versions

Windows 10.0.17763.0 , Android 12

**Comment by kristinx0211** [comment]
@macjohson can you upload a reproduction to github please?

**Comment by macjohson** [comment]
> @macjohson can you upload a reproduction to github please?

look at Controls, Convert and MainPage

[IconFontTest.zip](https://github.com/dotnet/maui/files/9032939/IconFontTest.zip)

**Comment by kristinx0211** [comment]
verified repro on windows with above repro project.

**Comment by mjfusa** [comment]
I have a customer that is also experiencing this.

**Comment by jinxinjuan** [comment]
Verified this issue with Visual Studio Enterprise 17.6.0 Preview 7.0. Repro on Android and windows platform with sample project. [IconFontTest.zip](https://github.com/dotnet/maui/files/9032939/IconFontTest.zip)

**Comment by jfversluis** [comment]
The strings are not formatted correctly to be used as unicode. Change all the &#x to \u and it will start working, at least with .NET 9 RC1 I have just tested this.

A cool helper is [this website](https://andreinitescu.github.io/IconFont2Code/) that lets you upload the icon font and gives you back the codes to use or even generated C# code with all these codes and identifiers.

![image](https://github.com/user-attachments/assets/8b20dc93-e2b8-4c36-a91a-9c5acdf8aa99)

Closing this as its not a bug.
